src/tests/functional/plugin/shared/include/low_precision_transformations/concat_transformation.hpp

```diff
@@ -34,7 +36,7 @@ class ConcatTransformation :
     public LayerTestsUtils::LayerTransformation {
 public:
     static std::string getTestCaseName(const testing::TestParamInfo<ConcatTransformationParams>& obj);
-    InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo &info) const override;
+    ov::test::utils::InputsMap get_input_map() override;
```

Comment:
there is `generate_inputs` method. Also you can use basic method to generate correct data per op (please check the implementation). I believe we should use default way to generate input data... (we use the same approach in conformance)
Could you please share the details why you decide to use `get_input_map()`???

Reply:
Discussed offline. This method should be removed. We will use static ranges for fake quintize in `generate_input`